src/util/util_errors.c

```diff
@@ -143,6 +143,8 @@ struct err_string error_to_str[] = {
     { "Error while parsing configuration file" }, /* ERR_INI_PARSE_FAILED */
     { "Failed to add configuration snippets" }, /* ERR_INI_ADD_SNIPPETS_FAILED */
 
+    { "Connect was interrupted"}, /* ERR_TLS_FAILURE */
```

Comment:
I would propose "TLS handshake" instead of "Connect".
"Connect" per se is usually done to this moment.